doc/sphinxext/docscrape.py

```diff
@@ -428,7 +429,11 @@ def __init__(self, func, role='func'):
             func, func_name = self.get_func()
             try:
                 # try to read signature
-                argspec = inspect.getargspec(func)
+                if 3 > sys.version_info[0]:
+                    argspec = inspect.getargspec(func)
+                else:
+                    # getargspec deprecated in python3
+                    argspec = inspect.signature(func)
```

Comment:
I would not change sphintext stuff manually. From a quick look at it it looks like all the files are coming from numpydoc and should be taken from a released version of numpydoc.